ark-testing/src/lib.rs

```diff
@@ -17,27 +17,38 @@ impl TestContext {
 
 	pub fn new(name: String, base_path: PathBuf) -> Self {
 		fs::create_dir_all(base_path.clone()).unwrap();
-		TestContext { name, datadir: base_path}
+		let context = TestContext { name, datadir: base_path};
+		context.init_logging().unwrap();
+		context
+	}
+
+	pub fn init_logging(&self) -> anyhow::Result<()> {
+		// We ignore the output
+		// An error is returned if the logger is initiated twice
+		// Note, that every test tries to initiate the logger
+		let _ = env_logger::builder().is_test(true).try_init();
+		Ok(())
```

Comment:
Hmm, this is going to be weird. cargo test suppresses stdout/err. We might have to do some special handling of cargo test (with flags etc to the extend it allows us to do that) or decide to not use cargo test. But let's try as long as it works for us :)